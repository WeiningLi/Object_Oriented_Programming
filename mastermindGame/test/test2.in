AA
Y
BB
Y
CC
Y
DD
Y
EE
N

