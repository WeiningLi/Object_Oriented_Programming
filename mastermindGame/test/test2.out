Welcome to Mastermind! Please enter your first guess:
2b,0w
You won in 1 guesses!
Would you like to play again? (Y/N): 
Welcome to Mastermind! Please enter your first guess:
0b,0w
You lost! The password was: DF
Would you like to play again? (Y/N): 
Welcome to Mastermind! Please enter your first guess:
0b,0w
You lost! The password was: BF
Would you like to play again? (Y/N): 
Welcome to Mastermind! Please enter your first guess:
0b,0w
You lost! The password was: EF
Would you like to play again? (Y/N): 
Welcome to Mastermind! Please enter your first guess:
0b,0w
You lost! The password was: FC
Would you like to play again? (Y/N): 
